src/components/Devotions/Devotions.js

```diff
@@ -0,0 +1,79 @@
+import React, {useState, useEffect, useContext, Fragment} from "react";
+import ReactDOM from 'react-dom';
+import {Input, Label, Button, Row, Col} from 'reactstrap';
+import {VoiceCommandsContext} from "../../context/VoiceCommandsContext";
+import AnalogClock from 'analog-clock-react';
+
+
+
+const Devotions = (props)=>{
+    const {SpeechRecognitionHook} = useContext(VoiceCommandsContext);
+    const [quote,setQuote] =  useState(null); //1
+
+// clock widget colors
+    let options = {
+        width: "300px",
+        border: true,
+        borderColor: "#2e2e2e",
+        baseColor: "#17a2b8",
+        centerColor: "#459cff",
+        handColors: {
+          second: "#d81c7a",
+          minute: "#fff",
+          hour: "#fff"
+        }
+    };
+
+    function useQuote() {
+        const [quote, setQuote] = React.useState(null)
+      
+        return quote
+      }
+
+    useEffect(()=>{
+        fetch("https://beta.ourmanna.com/api/v1/get/?format=text&order=random")
+        .then(response => response.json())
+        .then(quotes =>{
+            const randomIndex = Math.floor(Math.random() * quotes.length);
+            setQuote(quotes[randomIndex]);
+        });
```

Comment:
You might want to extract this into it's own function so that you can call it manually as well. You could set it up like this:

1. Define a function to get a random quote from the API
```
const getRandomQuote = () => {
        fetch("https://beta.ourmanna.com/api/v1/get/?format=text&order=random")
        .then(response => response.json())
        .then(quotes =>{
            const randomIndex = Math.floor(Math.random() * quotes.length);
            return quotes[randomIndex];
        });
}
```
2. In the useEffect() initial render call, you can use that function
```
useEffect(() => {
   let randomQuote = getRandomQuote();
   setQuote(randomQuote);
}, []);
```
3. Now you'll have the functionality in place to keep getting random quotes from the API, for example on a button press, or on a timer.

Reply:
The https://beta.ourmanna.com/api/v1/get/?format=text&order=random returns just one quote every time when you call it and returns a string, try this: 
```
    useEffect(() => {

        axios.get("https://beta.ourmanna.com/api/v1/get/?format=text&order=random").then(res => {
            const quote = res.data;
            console.log(quote )
        }).catch((err) => console.log(err));

    }, []);
```

the ```const quote``` is your string with a quote 

in order to get a JSON from the API you need to change url to https://beta.ourmanna.com/api/v1/get/?format=json&order=random